pkg/apis/sedna/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1078,6 +1078,267 @@ func (in *ModelStatus) DeepCopy() *ModelStatus {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *ObjectSearchService) DeepCopyInto(out *ObjectSearchService) {
```

Comment:
It's strange here, why is CI test OK if this zz.go didn't update?

Reply:
Added an issue #163

Reply:
I will fix #163 in a day or two, so remove this file since it has no relationship with this pr.

Reply:
ok